portfolio_home.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'>
+	<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <title>Jake Rooms - Portfolio</title>
+	<link rel='stylesheet' href='normalize.css'/>
+	<link rel='stylesheet' href='portfolio.css'/>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+	<main>
+		<!--Html for the top navigation bar-->
+		<header class='link_bar'>
+			<a class= 'top_link' href='portfolio_aboutme.html'>About me</a>
+		</header>
+		<!--Html for the image and project navigation bar-->
+		<section class='section_home'>
+			<img class='img_home' src='jake_face.jpg' alt=''/>
```

Comment:
Add alt info :)

Reply:
I think that leaving it blank is fine for a display image